requirements_for_test.txt

```diff
@@ -5,3 +5,4 @@ coverage==4.2b1
 pytest-cov==2.3.0
 jsonschema==2.5.1
 pylint==1.6.4
+assertpy==0.10
```

Comment:
The pros for me were the DSL made it more readable and the API has some useful functions https://github.com/ActivisionGameScience/assertpy